.travis.yml

```diff
@@ -1,7 +1,5 @@
 language: Java
 jdk:
   - oraclejdk8
-install: mvn -q clean -DskipTests -DskipITs install
-script: mvn -q clean verify
-after_success:
-  - mvn -q clean cobertura:cobertura-integration-test coveralls:report
+install: mvn -q clean -DskipTests -DskipITs install && mvn -q clean cobertura:cobertura-integration-test coveralls:report
```

Comment:
This is running the cobertura tests and report twice. Once in the install and once in the script.  The install should only download dependencies.  So I would remove the cobertura from the install step.